### PR TITLE
Pop up using Javascript #PR

### PR DESCRIPTION
**Pull request by Sonikak004**
### In this Branch I have :

- when button to check project details is clicked, the popup with details about the project appears ✔️ 
- when the close (X) button is clicked,, the popup disappears. ✔️ 
- used JavaScript array of objects to store all of the information for all projects ✔️ 
   - name, description, featured image, technologies, link to live, link to source 
- updated the main page so that the projects section is created dynamically 💯 
-  implemented the popup window 👍 
- No linter Errors ✅

### CODE REVIEW

**Review by Qoosim — COMMENTED** [review]
#  *Status: Invalid Request* :red_circle: 

Hi @team, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that 

- [ ] has Microverse template logo which has been to be removed and replace with student personal logo if he or she has one. Kindly remove the Microverse logo and replate it with your logo or leave empty.
- [ ] The link to the MIT license is broken, kindly fix that too by following this [article](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to properly set it up for your project.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -1,367 +1,284 @@
...
+              <a class="button-text">See Project</a>
+            </button>
+          </div>
+        </div>
+      </section>
+        <!-- professional art cards section -->
+        <div class="project-cards">
+         <section class="professional-art" id="project-section">
+        </section> 
+     </div>
```
Comment (emmiiorji) [inline index.html:151]: [Optional] Well done for rendering the projects dynamically from JavaScript. As given on the figma design, on desktop version, the project cards look a little different. The card without the "See Project" button shows how the cards would display normally. However, the card having the "See Project" button shows how the cards display when hovered.

Normal project card:
image

Project card when hovered:
image

File: index.html [hunk 2]
```diff
@@ -1,367 +1,284 @@
...
-          <h2 class="contact-me-para">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+      <section class="about-me" id="about-section">
+        <div class="about-me-div">
+          <h2 class="about-me-text">About me</h2>
+          <p class="about-me-para">Hello I'm a software developer! I can help you build a product , feature or website
+            Look through some of my work and experience! If you like what you see and have a project you need coded,
+            don't hestiate to contact me.</p>
+
+          <!-- get my resume button -->
+          <button class="button3" type="button">
```
Comment (emmiiorji) [inline index.html:163]: [Optional] The website seems to have a horizontal scroll. For a better user experience, I think you should avoid having a horizontal scroll.

image

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @Sonikak004, 
# Status: Changes Required ♻️

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

BbcanBbcan3GIF


## Good Highlights✔️

- Linter checks are passing
- Good commit messages
- Good use of GitHub flow


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js [hunk 3]
```diff
@@ -0,0 +1,563 @@
...
+  {
+    id: 0,
+    name: 'Keeping track of hundreds  of components website',
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+    image: './images/probackground.png',
+    button: ['see live', 'see source', 'see project'],
+    technologies: ['html', 'css', 'Bootstrap', 'Ruby'],
+    liveLink: 'https://sonikak004.github.io/',
+    sourceLink: 'https://github.com/Sonikak004/my-portfolio',
+  },
```
[the remark this answers is not in the document]
Comment (GzimAsani) [reply index.js:102]: - [ ] You have a lot of repetition on your CSS file, and this falls under the category of bad CSS practice. For example, you have used display:flex 46 times. I would suggest you create a class d-flex and use display:flex there and use on your HTML whenever you need. Also make sure to take into the account the previous reviewer's request based on css repetition regarding to ptext 1,2,3,4 etx
image

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello @Sonikak004                 :wave:   
 Good job so far,  done There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job with fixing the mobile version as the previous reviewer requested :+1: 

 ## Required Changes 
- [ ] Check the comments below :arrow_double_down: 


 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._









 ### Hello @developertariq                :wave:   
 Good job so far,  done There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job with the redux and styling :+1: 

 ## Required Changes 
- [ ] Check the comments below :arrow_double_down: 


 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Deepakdanger — APPROVED** [review]
Hi @Sonikak004, 

### Status: Approved ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**